sycl/include/CL/sycl/detail/image_accessor_util.hpp

```diff
@@ -945,19 +931,19 @@ DataT ReadPixelDataLinearFiltMode(const cl_int8 CoordValues,
 
   // Get Color Values at each Coordinate.
   cl_float4 Ci0j0k0 = getColorInFloat(cl_int4{i0, j0, k0, 0});
-  
```

Comment:
Do you really need these stylistic changes in in this pull-request?

Reply:
I think, no. My editor does it automatically. I'll revert these.

Reply:
Done.